### PR TITLE
upgrading to "d2l-colors" 2.0.0 and prefixing files with "d2l-"

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

File: menu-item-styles.html [hunk 1]
```diff
@@ -1,14 +1,13 @@
 <link rel="import" href="../polymer/polymer.html">
-<link rel="import" href="../vui-colors/colors.html">
+<link rel="import" href="../d2l-colors/d2l-colors.html">
 
 <dom-module id="d2l-menu-item-styles">
 	<template>
-		<style include="vui-colors"></style>
-		<style include="vui-small-text"></style>
```
Comment (dlockhart) [inline menu-item-styles.html]: I was getting a console warning about this one -- doesn't appear to be needed.

**Comment by dbatiste** [comment]
Looks good.  I am guessing you'll have a similar PR for typography update?

**Comment by dlockhart** [comment]
Yeah there are going to be a whole batch of them... it's all or nothing with Bower.

File: menu-item-styles.html (hunk 1 above)
Comment (dbatiste) [inline menu-item-styles.html]: Cool thanks.  Looks like I forgot to remove this when I recently updates styles to better reflect design.

**Pull request merged**